apps/aechannel/src/aesc_create_tx.erl

```diff
@@ -153,7 +155,8 @@ process(#channel_create_tx{} = Tx, Trees, Env) ->
           lock_period(Tx),
           fee(Tx),
           nonce(Tx),
-          round(Tx)),
+          round(Tx),
+          ChannelPubkey),
```

Comment:
This is independent refactoring and not strictly needed, right?

It is ok to keep here - I am asking simply because I am considering regression testing across tags using this module as interface.

Reply:
Without this, the wrong `channel_id` event is being sent for GAs - if the `initiator` is a generalized account one's nonce is not being used in `channel_id` computation. This fixes it